worker.transport.js

```diff
@@ -0,0 +1,8 @@
+const { parentPort } = require("node:worker_threads");
+const { makeNodeTransport } = require("@sentry/node");
+
+let transport = makeNodeTransport({});
+
+parentPort.on("message", (message) => {
+  transport.send(envelope);
```

Comment:
Do we have to make envelope -> message here?

Reply:
Oh yes, we def should.